contract/wizards/contract_manually_create_invoice.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class ContractManuallyCreateInvoice(models.Model):
```

Comment:
why not `TransientModel` and you save the extra permission?

Reply:
I would like to avoid that this action is available for all accounting users.

Reply:
Do that other way, adding a group on a menu or a button, but not this way.

Reply:
another advantage of making it a Model is to keep track for all manual runs in the database.

Reply:
I don't see that as any advantage, as that's not browseable. If you want that, put a log entry in the chatter of the contract (with, in other hand, would be great for traceability in general))

Reply:
@pedrobaeza , I applied you suggestion.

Please review my last commit.